test/fixture/terraform_15_1.0/git_issue_88__corrupt_hcl_file/expected/main.terratag.tf

```diff
@@ -0,0 +1,31 @@
+provider "aws" {
+  version = "~> 2.0"
+  region  = "us-east-1"
+}
+
+resource "aws_s3_bucket" "b" {
+  bucket = "my-tf-test-bucket"
+  acl    = "private"
+
+  tags = merge(tomap({
+    "Name" = "My bucket"
+  }), local.terratag_added_main)
```

Comment:
This is still WIP. The test will fail because of this line.
It will modify this line and apply a merge to the merge.